bin/cheribsdtest/cheribsdtest_fault.c

```diff
@@ -109,6 +109,10 @@ test_fault_perm_seal(const struct cheri_test *ctp __unused)
 		cheribsdtest_failure_err("sysctlbyname(security.cheri.sealcap)");
 	sealcap = cheri_andperm(sealcap, ~CHERI_PERM_SEAL);
 	sealed = cheri_seal(ip, sealcap);
+#if !CHERI_SEAL_VIOLATION_EXCEPTION
+	if (!cheri_gettag(sealed))
+		cheribsdtest_success();
+#endif
```

Comment:
`#else`?

Reply:
The cheribsdtest_failure_errx is both correct if the tag isn't stripped and apparently needed to keep the cap live.

Reply:
Then I guess update the comment to change faulting to something else? Though I guess that's part of the larger problem of having these tests be `test_fault_*` in cheribsdtest_fault.c...

Reply:
I'll just remove the comment. It's obvious that it's needed if the non-faulting case fails and obviously harmless even if the status would be ignored in the faulting case.